include/mbedtls/base64.h

```diff
@@ -89,6 +89,16 @@ int mbedtls_base64_self_test( int verbose );
 
 #endif /* MBEDTLS_SELF_TEST */
 
+#if defined(MBEDTLS_TEST_HOOKS)
```

Comment:
Why not create `library/base64_invasive.h` for this, just as we already have `library/ssl_invasive.h` in the same branch? Considering 2.x is not LTS yet, I don't see a reason not to create new files.